### DISCUSSION TITLE
コラボレーターへの招待

### DISCUSSION THREAD

**Discussion by PrefKarafuto** [PrefKarafuto/ex0ch#20] [Announcements]
コラボレーターに招待した方若しくは参加希望の方はこちら

**Comment by PrefKarafuto** [comment]
招待を送った方です

**Reply by PrefKarafuto** [reply]
[GeneralYowaimo](https://github.com/GeneralYowaimo)氏

**Comment by PrefKarafuto** [comment]
てす

**Reply by stat-tools** [reply]
よろしくお願いします🙋

**Reply by pentatomoidea** [reply]
よろしくお願いします🥰

**Reply by koushinkyouto** [reply]
よろしくお願いします

**Comment by ghost** [comment]
そのPerl使ったことない人です、
その、Perlはじめの人ですが、そのどこから学べばいいですか？
掲示板作りたいです

**Reply by PrefKarafuto** [reply]
掲示板作りたいなら、PHPの方が（少なくともPerlよりは）将来性もあるのでそっち勉強したほうがいいと思いますね
